scripts/verify.sh

```diff
@@ -26,5 +26,8 @@ elif [[ $1 == "lint" ]]
         echo "Linting Checker..."
         launch_pylint ./src/checkers
         r2=$?
-        exit $((r1 + r2))
+        echo $((r1 + r2))
 fi
+
```

Comment:
If you want to add a windows command please add these to a separate file, something like `verify.ps1`.

Reply:
I've added a file called verifyWindows.sh with these changes, and left verify.sh as is. I'll add docs to the wiki page if this change goes through

Reply:
Because I don't want to have to change other parts of the the script. It doesn't seem worth the effort.